### PR TITLE
Update openAlias,OpenUser

### PR DESCRIPTION
**Pull request by sangdo-lee**

### CODE REVIEW

File: src/main/java/com/rapid7/client/dcerpc/mssamr/SecurityAccountManagerService.java [hunk 1]
```diff
@@ -73,12 +73,12 @@ public UserHandle openUser(DomainHandle domainHandle, int sid) throws IOExceptio
 	return response.getHandle();
     }
 
-    public AliasHandle openAlias(DomainHandle domainHandle, int sid) throws IOException {
+    public AliasHandle openAlias(UserHandle userHandle, int sid) throws IOException {
```
Comment (bli-r7) [inline src/main/java/com/rapid7/client/dcerpc/mssamr/SecurityAccountManagerService.java]: This should be the DomainHandle not UserHandle

Comment (sangdo-lee) [reply src/main/java/com/rapid7/client/dcerpc/mssamr/SecurityAccountManagerService.java]: yep!

File: src/main/java/com/rapid7/client/dcerpc/mssamr/messages/SamrOpenLocalGroupRpcRequest.java [hunk 2]
```diff
@@ -21,21 +21,21 @@
 import java.io.IOException;
 import com.rapid7.client.dcerpc.io.PacketOutput;
 import com.rapid7.client.dcerpc.messages.RequestCall;
-import com.rapid7.client.dcerpc.mssamr.objects.DomainHandle;
+import com.rapid7.client.dcerpc.mssamr.objects.UserHandle;
 
 public class SamrOpenLocalGroupRpcRequest extends RequestCall<SamrOpenLocalGroupRpcResponse> {
```
Comment (bli-r7) [inline src/main/java/com/rapid7/client/dcerpc/mssamr/messages/SamrOpenLocalGroupRpcRequest.java]: OP_NUM 27 should be SamrOpenAlias

https://msdn.microsoft.com/en-us/library/cc245751.aspx

File: src/main/java/com/rapid7/client/dcerpc/mssamr/messages/SamrOpenUserResponse.java [hunk 3]
```diff
@@ -21,13 +21,11 @@
...
 
-public class SamrOpenUserResponse extends HandleResponse {
+public class SamrOpenUserResponse extends HandleResponse<UserHandle> {
 
     @Override
-    public UserHandle getHandle() {
-        UserHandle handle = new UserHandle();
-        handle.setBytes(super.handle.getBytes());
-        return handle;
+    public UserHandle initHandle() {
```
Comment (bli-r7) [inline src/main/java/com/rapid7/client/dcerpc/mssamr/messages/SamrOpenUserResponse.java]: protected

File: src/main/java/com/rapid7/client/dcerpc/mssamr/messages/SamrOpenAliasResponse.java [hunk 4]
```diff
@@ -21,13 +21,11 @@
...
 
-public class SamrOpenLocalGroupRpcResponse extends HandleResponse {
+public class SamrOpenAliasResponse extends HandleResponse<AliasHandle> {
 
     @Override
-    public AliasHandle getHandle() {
-        AliasHandle handle = new AliasHandle();
-        handle.setBytes(super.handle.getBytes());
-        return handle;
+    public AliasHandle initHandle() {
```
Comment (bli-r7) [inline src/main/java/com/rapid7/client/dcerpc/mssamr/messages/SamrOpenAliasResponse.java]: protected

File: src/main/java/com/rapid7/client/dcerpc/mssamr/messages/SamrOpenAliasRequest.java [hunk 5]
```diff
@@ -23,19 +23,19 @@
...
 
     private final DomainHandle handle;
     private final int userRid;
     private final int desiredAccess;
 
-    public SamrOpenLocalGroupRpcRequest(DomainHandle handle, int userRid) {
-        // SAMR_ALIAS_ACCESS_LOOKUP_INFO(8)
-        this(handle, userRid, 8);
+    public SamrOpenAliasRequest(DomainHandle handle, int userRid) {
+	// SAMR_ALIAS_ACCESS_LOOKUP_INFO(8)
```
Comment (bli-r7) [inline src/main/java/com/rapid7/client/dcerpc/mssamr/messages/SamrOpenAliasRequest.java]: indentation should be 4

**Pull request merged**